<launch>
  <node
    pkg = "autorally_vision"
    type = "aruco_detector"
    name = "aruco_detector_node"
    output = "screen">



    <param name="image_topic" value="/left_camera/image_raw" />

    <param name="adaptiveThreshConstant" value="7" />
    <param name="adaptiveThreshWinSizeMax" value="23" />
    <param name="adaptiveThreshWinSizeMin" value="3" />
    <param name="adaptiveThreshWinSizeStep" value="10" />
    <param name="aprilTagCriticalRad" value="/left_camera/image_raw" />          <!--default?-->>
    <param name="aprilTagDeglitch" value="0" />


  </node>

</launch>

